### PR TITLE
feat(ToyCar): Add car animation depending on the hinge angle value and add Uno Platform theme

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
Add car animation depending on the hinge angle value.
Moving the cars in the middle of the device if the angle is equal to 180 degrees.
Also moving them to the right side of the device if the angle is equal to 360 degrees meaning that both screens are facing aware of each other and the device is fully opened. In addition, At the same time, starting the car animation and the scene animation in the background.

Testing the hinge angle with the Surface Duo emulator: https://devblogs.microsoft.com/surface-duo/surface-duo-testing-tips-tricks/#hinge-angle

ToyCar-Android-SurfaceDuo-HingeAngle

### CODE REVIEW

File: UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj [hunk 1]
```diff
@@ -47,9 +51,12 @@
     <AndroidLinkMode>SdkOnly</AndroidLinkMode>
     <AndroidCreatePackagePerAbi>true</AndroidCreatePackagePerAbi>
     <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
-    <AotAssemblies>true</AotAssemblies>
-    <EnableLLVM>true</EnableLLVM>
+    <AotAssemblies>false</AotAssemblies>
```
Comment (MatFillion) [inline UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj:54]: was it intentional to remove AOT?

Comment (agneszitte-nventive) [reply UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj:54]: No I think it was changed by going to the properties and I didn't notice the changes at first.
I will update the .csproj thanks ;) !

Comment (jeromelaban) [reply UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj:54]: It changes back by itself in some editions of VS.

**Review by jeromelaban — APPROVED** [review]

File: UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj (hunk 1 above)
Comment (agneszitte-nventive) [reply UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj:54]: Yes that's what I noticed

Comment (agneszitte-nventive) [reply UI/ToyCar/ToyCar.Droid/ToyCar.Droid.csproj]: Update done thanks !

**Pull request merged**